sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallingServerClientBuilder.java

```diff
@@ -279,20 +283,25 @@ public CallingServerClientBuilder clientOptions(ClientOptions clientOptions) {
         return this;
     }
 
-    private HttpPipelinePolicy createHttpPipelineAuthPolicy() {
+    private List<HttpPipelinePolicy> createHttpPipelineAuthPolicies() {
```

Comment:
return types and method parameters should use interface types. This should be IList<>

Reply:
This is Java, List is the interface :)

Reply:
Sorry, just realized this is Java. Check to see if they return List from other methods.

Reply:
Yep, I was using ArrayList as the concrete class. I see List in some method signatures for other areas of the calling server sdks (such as join call).